src/gluonts/kernels/_kernel_output.py

```diff
@@ -97,7 +97,7 @@ def get_args_proj(self, float_type: DType = np.float32) -> ArgProj:
         return ArgProj(
             args_dim=self.args_dim,
             domain_map=gluon.nn.HybridLambda(self.domain_map),
-            float_type=float_type,
+            dtype=float_type,
```

Comment:
`dtype=dtype`?

Reply:
This is for the next PR as it triggers many changes.